rhizome/host/bin/prep_host.rb

```diff
@@ -61,6 +61,7 @@
   r "curl -L3 -o cloud-hypervisor #{CloudHypervisor::VERSION.url.shellescape}"
   FileUtils.chmod "a+x", "cloud-hypervisor"
 end
+CloudHypervisor::NEW_VERSION.download
```

Comment:
In general...if we're going through the trouble to name things `fooLegacy`, can't you spare yourself trouble by _not_ naming the new version `new`? Anything without `*Legacy` in it can presumed to be current, right?

Reply:
renamed the original version (as well as existing references) to `VERSION_LEGACY`. `NEW_VERSION` was only meant to be a temporary name until the legacy version is removed.